protocol/innpv.proto

```diff
@@ -250,6 +260,24 @@ message HardwareInfo {
   repeated string gpus = 3;
 }
 
+message UtilizationNode {
+  // node representation of a slice in perfetto (pytorch profiler)
+  uint32 slice_id = 1;
+  string name = 2;
+  uint64 start = 3;
+  uint64 end = 4;
+  uint64 cpu_forward = 5;
+  uint64 cpu_forward_span = 6;
```

Comment:
Do these have to have integral types?

Reply:
yeah, well the times are all in nanoseconds that's why I am using uint64